easybuild/easyconfigs/c/cairo/cairo-1.14.6-intel-2016a.eb

```diff
@@ -24,6 +24,13 @@ dependencies = [
     ('fontconfig', '2.11.94'),
     ('expat', '2.1.0'),
     ('GLib', glibver),
+    ('libX11', '1.6.3'),
+    ('libxcb', '1.11.1'),
+    ('libXrender', '0.9.9'),
+    ('libXext', '1.3.3'),
+]
+builddependencies = [
```

Comment:
We don't have strict guidelines for this imho..

I think you'll find all combos w.r.t. order and empty line in between or not.

Let's change this consistently once we have an automated check for it.